zilencer/models.py

```diff
@@ -84,6 +84,35 @@ def __str__(self) -> str:
         return f"{self.server!r} {self.user_id}"
 
 
+class RemoteRealm(models.Model):
+    """
+    Each object corresponds to a single remote Realm that is using the
+    Mobile Push Notifications Service via `manage.py register_server`.
+    """
+
+    server = models.ForeignKey(RemoteZulipServer, on_delete=models.CASCADE)
+
+    # The unique UUID and secret for this realm.
+    uuid = models.UUIDField(unique=True)
```

Comment:
Do we want a `realm_id` column here? The reason it might be handy is to associate the `RemoteRealmCount` objects' `realm_id` column with. I'm not sure it really is necessary, in that the theory would be that we'd hook those up via a new `RemoteRealmCount.remote_realm` column, and whenever we're writing new rows, we should know what `RemoteRealm` to associate it with, via the metadata in our API request.

Reply:
Yeah  it seems redundant. A more natural way would be to have `realm_id` on the `RemoteRealm` actually, and then the `RemoteRealmCount` objects would figure out this out through the appropriate JOIN on `.remote_realm`, right? But I guess we can't do that because e.g. old servers will not have `RemoteRealm` entries to link to from their `RemoteRealmCount`s

Reply:
Yeah, exactly, there's no `RemoteRealm` for older servers. (Perhaps we could create them ... but we'd be making up most of the fields)